MSFragger-GUI/src/com/dmtavt/fragpipe/tools/skyline/WriteTransitionList.java

```diff
@@ -0,0 +1,138 @@
+package com.dmtavt.fragpipe.tools.skyline;
+
+import com.dmtavt.fragpipe.util.UnimodOboReader;
+
+import java.io.*;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static com.dmtavt.fragpipe.cmd.ToolingUtils.UNIMOD_OBO;
+import static com.dmtavt.fragpipe.cmd.ToolingUtils.getUnimodOboPath;
+import static com.dmtavt.fragpipe.util.Utils.AAMasses;
+
+public class WriteTransitionList {
+
+    public static Map<String, Float> unimodMasses;
+    private static HashMap<String, Integer> columns;
+    private static final double AVERAGINE_ISOTOPE = 1.00235;
```

Comment:
I think 1.00235 is the better value to use for peptides, since it is closer to the actual spacing of isotopic peaks in spectra (1.003355 is what it would be if peptides were made up of pure carbon). Ideally we'd change all the tools to 1.00235, but maybe it's not worth the effort/testing. I guess either is probably fine for Skyline, since it is only 1 mDa and the difference will almost always be within the match tolerance